docs/programming-oak.md

```diff
@@ -97,8 +97,8 @@ impl oak::grpc::OakNode for Node {
         oak_log::init_default();
         Node
     }
-    fn invoke(&mut self, method: &str, req: &[u8], out: oak::WriteHandle) {
-        dispatch(self, method, req, out)
+    fn invoke(&mut self, stream_id: i32, method: &str, req: &[u8], out: oak::WriteHandle) {
```

Comment:
It seems we should just pass a parameter `out` of type `ResponseWriter` to this, rather than a `stream_id` and a `WriteHandle`.

Reply:
Good idea, done &ndash; that will also make any changes for #285 smaller too.